api/v1beta1/openstackbaremetalset_types.go

```diff
@@ -30,7 +30,7 @@ type OpenStackBaremetalSetSpec struct {
 	Replicas int `json:"replicas,omitempty"`
 	// Remote URL pointing to desired RHEL qcow2 image
 	RhelImageURL string `json:"rhelImageUrl,omitempty"`
-	// ProvisionServerName Optional. If supplied will be used as the base Image for the baremetalset instead of RhelImageURL.
+	// OpenStackProvisionServerName Optional. If supplied will be used as the base Image for the baremetalset instead of RhelImageURL.
 	ProvisionServerName string `json:"provisionServerName,omitempty"`
```

Comment:
Looks like this needs to be changed from `ProvisionServerName` to `OpenStackProvisionServerName`, given the comment above

Reply:
I updated the comment so this stays the same (ProvisionServerName).